velox/exec/tests/utils/AssertQueryBuilder.h

```diff
@@ -101,15 +101,16 @@ class AssertQueryBuilder {
 
   /// Run the query and collect all results into a single vector. Throws if
   /// query returns empty result.
-  RowVectorPtr copyResults(memory::MemoryPool* pool);
+  RowVectorPtr copyResults(memory::MemoryPool* FOLLY_NONNULL pool);
 
  private:
   std::pair<std::unique_ptr<TaskCursor>, std::vector<RowVectorPtr>>
   readCursor();
 
+  DuckDbQueryRunner* FOLLY_NONNULL const duckDbQueryRunner_;
```

Comment:
This can be null, hence, FOLLY_NULLABLE ?

Reply:
I see. Thanks!